gradle-plugin/README.md

```diff
@@ -396,15 +442,30 @@ emerge {
     // The build variants Reaper is enabled for.
     // When Reaper is enabled the application bytecode will be instrumented to support Reaper.
     enabledVariants.set(listOf("release", "releaseVariant2"))
-    // The key used to identify Reaper reports for your organization. Emerge recommends setting this as an environment variable
-    // Note: This key is not the same as the API key used for uploading to Emerge - you can find this
-    publishableApiKey.set(System.getenv("REAPER_API_TOKEN"))
+    // The key used to identify Reaper reports for your organization. Emerge recommends setting this as an environment variable.
+    // This key can be found at https://www.emergetools.com/settings?tab=feature-configuration&cards=reaper_enabled
+    // Note: This key is not the same as the API token used for uploading to Emerge.
+    publishableApiKey.set(System.getenv("REAPER_API_KEY"))
 
     // Optional, defaults to 'release'
     tag.set("release")
     // Alternatively, use `setFromVariant()` to set the tag from the Android build variant name
     tag.setFromVariant()
+  }
 
+  distribution {
+    // The build variants Distribution is enabled for.
+    enabledVariants.set(listOf("release", "staging"))
+    // The key used to authenticate downloads for future updates.
+    // Emerge recommends setting this as an environment variable.
+    // This key can be found at https://www.emergetools.com/settings?tab=feature-configuration&cards=distribution_enabled
+    // Note: This key is not the same as the API token used for uploading to Emerge.
+    apiKey.set(System.getenv("DISTRIBUTION_API_KEY"))
+
+    // Optional, defaults to 'release'
+    tag.set("release")
```

Comment:
Tag doesn't seem to be relevant currently - I'd suggest removing it until we have an upload-related task around distro to avoid confusion

Reply:
Tag is used to define the release 'channel' (when the SDK checks for an update we send the tag and only consider builds a with matching tag). Possibly we should have a distribution upload task but I was worried that just makes it more confusing.